gpd/gpd-pocket/default.nix

```diff
@@ -0,0 +1,87 @@
+{ lib, pkgs, ... }:
+
+# GPD Pocket support
+#
+# Most of the configuration comes from the Arch Linux Wiki:
+# https://wiki.archlinux.org/index.php/GPD_Pocket
+#
+# Missing:
+# - media keys
+# - brightness control
+# - video over USB-C
+
+{
+  imports = [
+    ../../common/cpu/intel
+    ../../common/pc/laptop
+  ];
+
+  # kernel
+  # TODO: use https://github.com/jwrdegoede/linux-sunxi?
+  boot = {
+    kernelParams = ["fbcon=rotate:1"];
+    kernelPackages = pkgs.linuxPackages_4_14;
+  };
+
+  # wifi
+  nixpkgs.config.allowUnfree = true;
+  hardware.firmware = with pkgs; [
+    brcmfmac4356-firmware
+  ];
+
+  # power management
+  services.tlp = {
+    enable = true;
+    extraConfig = ''
+      DISK_DEVICES="mmcblk0"
+      DISK_IOSCHED="deadline"
+    '';
+  };
+
+  hardware.pulseaudio = {
+    enable = true;
```

Comment:
So far there are no profiles that explicitly depend on PulseAudio: the idea behind that is that people that don't want to use PulseAudio (i.e. pure ALSA setup, or Jack) are not excluded. Extra configuration for PulseAudio (to make it work once it's enabled) is welcome, though.